consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/model/claimdefences/ClaimDefencesResponse.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.moj.sdt.cmc.consumers.model.claimdefences;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ClaimDefencesResponse {
+    private Integer resultCount;
+    private ClaimDefencesResult[] results;
```

Comment:
you can use List, instead of Array, Jackson libraries will handle the coversion

Reply:
Done